debian/copyright

```diff
@@ -1,5 +1,35 @@
-Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: revpi-wallpaper
+Upstream-Contact: KUNBUS GmbH <support@kunbus.com>
+Source: https://github.com/RevolutionPi/revpi-wallpaper
 
-Files: *
-Copyright: 2016-2021 Kunbus GmbH
-License: proprietary
+Files:     *
+Copyright: 2016-2023 Kunbus GmbH
+License:   CC-BY-SA-4.0
+
+Files:     debian/*
+Copyright: 2022-2023 KUNBUS GmbH
+License:   GPL-2+
+
+License:   GPL-2+
+ This package is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2 of the License, or
+ (at your option) any later version.
+ .
+ This package is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+ .
+ You should have received a copy of the GNU General Public License
+ along with this program. If not, see <http://www.gnu.org/licenses/>
+ .
+ On Debian systems, the complete text of the GNU General
+ Public License version 2 can be found in "/usr/share/common-licenses/GPL-2".
+
+License: CC-BY-SA-4.0
+ This work is licenced under the Creative Commons Attribution-Share Alike 4.0
```

Comment:
You need to add the full license text here.
See /usr/share/doc/gnome-weather/copyright for an example

Reply:
Interesting. This was taken from an official debian gnome package: https://sources.debian.org/src/gnome-devel-docs/40.3-1/debian/copyright/

Reply:
Interesting, it might be good enough. But to be on the safe side I would include the full text. Otherwise it would be possible to change the license text on the web...
And we should also include it as license in the master branch.